Examples/Movies/SearchScreen.js

```diff
@@ -13,34 +13,32 @@
  *
  * @flow
  */
-'use strict';
 
-var React = require('react-native');
-var {
+import React, {
   ActivityIndicatorIOS,
+  Component,
   ListView,
   Platform,
   ProgressBarAndroid,
   StyleSheet,
   Text,
   View,
-} = React;
-var TimerMixin = require('react-timer-mixin');
+} from 'react-native';
 
-var invariant = require('invariant');
-var dismissKeyboard = require('dismissKeyboard');
+import invariant from 'invariant';
+import dismissKeyboard from 'dismissKeyboard';
 
-var MovieCell = require('./MovieCell');
-var MovieScreen = require('./MovieScreen');
-var SearchBar = require('SearchBar');
+import MovieCell from './MovieCell';
+import MovieScreen from './MovieScreen';
+import SearchBar from './SearchBar';
```

Comment:
I think this is a good move, from @providesModule to relative import. I don't think we want to encourage providesModule use in consumers' code.